webcompat/static/js/lib/issue-wizard-bugform.js

```diff
@@ -798,6 +803,10 @@ BugForm.prototype.nextStep = function(e) {
       this.blockNext = false;
     }
 
+    if (trigger.val() === this.otherProblemId) {
+      hideStep = 4;
+    }
```

Comment:
This condition is part of `BugForm.prototype.nextStep` method that is quite big and contains lots of logic in it. Could you please split the method into smaller functions ?